aztk/clusterlib.py

```diff
@@ -359,7 +366,7 @@ def pretty_dedicated_node_count(self, cluster: ClusterModel)-> str:
             return '{}'.format(cluster.dedicated_nodes)
 
     def pretty_low_pri_node_count(self, cluster: ClusterModel)-> str:
-        if (cluster.pool.allocation_state is batch_models.AllocationState.resizing\
```

Comment:
was `\` required?

Reply:
No -- not required, just a formatting ting. Right above this, pretty_dedicated_node_count() does the same thing without a backslash.